src/embind/embind.js

```diff
@@ -180,18 +180,16 @@ var LibraryEmbind = {
     return errorClass;
   },
 
-  $createNamedFunction: (name, body) => Object.defineProperty(body, 'name', {
-    value: name
-  }),
+  $createNamedFunction: (name, func) => Object.defineProperty(func, 'name', { value: name }),
```

Comment:
This was just a driveby fix change the name of the second argument to something more accurate.